app/Notifications/Training/WaitingListRemovalAdded.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace App\Notifications\Training;
+
+use App\Notifications\Notification;
+use Carbon\Carbon;
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Notifications\Messages\MailMessage;
+
+class WaitingListRemovalAdded extends Notification implements ShouldQueue
+{
+    use Queueable;
+
+    private $list_name;
+    private $removal_date;
+
+    /**
+     * Create a new notification instance.
+     *
+     * @return void
+     */
+    public function __construct(String $list_name, Carbon $removal_date)
+    {
+        parent::__construct();
+
+        $this->list_name = $list_name;
+        $this->removal_date = $removal_date;
+    }
+
+    /**
+     * Get the notification's delivery channels.
+     *
+     * @param  mixed  $notifiable
+     * @return array
+     */
+    public function via($notifiable)
+    {
+        return ['mail', 'database'];
```

Comment:
```suggestion
        return ['mail'];
```
You haven't defined an array representation, so should be via mail.

Reply:
Thanks Callum, I wasn't entirely sure what the purpose of this was and aired on the side of caution when referencing similar classes. I will change this.